pandas/core/groupby.py

```diff
@@ -191,6 +191,63 @@
     dtype: int64
     """)
 
+_numeric_operations_doc_template = """
+    Compute %(f)s of group values.
+
+    For multiple groupings, the result index will be a
+    :class:`~pandas.MultiIndex`.
+
+    Parameters
+    ----------
+    kwargs : dict
+        Optional keyword arguments to pass to `%(f)s`.
+
+    Returns
+    -------
+    Series or DataFrame
+
+    {examples}
+    {see_also}
+    """
+
+_numeric_operations_examples = dict(
+    max="""Examples
+    --------
+    Grouping by one column.
+
+    >>> df = pd.DataFrame({'A': 'a b b b'.split(), 'B': [1,2,2,3], 'C': [4,5,6,7]})
```

Comment:
The [Conventions for the examples](https://python-sprints.github.io/pandas/guide/pandas_docstring.html#conventions-for-the-examples) in the guidelines states:

> For more complex examples (groupping for example), avoid using data without interpretation, like a matrix of random numbers with columns A, B, C, D… And instead use a meaningful example

Therefore, this examples should be improved.